pkg/controller/disruption/disruption.go

```diff
@@ -345,7 +347,7 @@ func (dc *DisruptionController) deletePod(obj interface{}) {
 func (dc *DisruptionController) enqueuePdb(pdb *policy.PodDisruptionBudget) {
 	key, err := controller.KeyFunc(pdb)
 	if err != nil {
-		glog.Errorf("Cound't get key for PodDisruptionBudget object %+v: %v", pdb, err)
+		utilruntime.HandleError(fmt.Errorf("Cound't get key for PodDisruptionBudget object %+v: %v", pdb, err))
```

Comment:
Hm, this error doesn't fall into the catch-all `HandleError` so that's probably ok. (Although I don't know if we will ever be able to handle this in a different way)